x-pack/plugins/console_extensions/spec/overrides/xpack.ml.revert_model_snapshot.json

```diff
@@ -0,0 +1,7 @@
+{
+  "xpack.ml.revert_model_snapshot": {
+    "data_autocomplete_rules": {
+      "delete_intervening_results": "__flag__"
```

Comment:
When `__flag__` is used as a query param value, e.g. in the xpack info endpoint, it will autosuggest true or false as values. In this case, it just completes the field with a `"__flag__"` string.